src/generator/AutoRest.Go/GoCodeGenerator.cs

```diff
@@ -60,7 +60,8 @@ public override void NormalizeClientModel(ServiceClient serviceClientModel)
         public override async Task Generate(ServiceClient serviceClient)
         {
             string packageName = GoCodeNamer.PackageNameFromNamespace(Settings.Namespace);
-            
+            string[] version = GoCodeNamer.SDKVersionFromPackageVersion(Settings.PackageVersion);
```

Comment:
If we make it optional then we need to put some default value in version like 0.0.0. There is a chance if we do that we might release SDK with wrong default version so I made it required to be sure what version we pass the version all the time. But if you still want I can make this optional?